### PR TITLE
Add cluster for a fallback test

### PR DESCRIPTION
**Pull request by eugeneo**
Trying to address this Kokoro failure:
```
+ [21:10:41 UTC]	 Installing packages with apt, see install-apt.log
+ [21:11:21 UTC]	 Fetching GKE cluster credentials
ERROR: (gcloud.container.clusters.get-credentials) One of [--location, --zone, --region] must be supplied.
```

### CODE REVIEW

File: .kokoro/psm_interop_kokoro_lib.sh [hunk 1]
```diff
@@ -206,9 +206,11 @@ psm::dualstack::run_test() {
 
 #######################################
 # Fallback Test Suite setup.
+#
+# This test does not actively need GKE but Kokoro fails if no cluster is provided.
 #######################################
 psm::fallback::setup() {
-  : # Placeholder for future setup commands
+  activate_gke_cluster GKE_CLUSTER_PSM_BASIC
```
Comment (XuanWang-Amos) [inline .kokoro/psm_interop_kokoro_lib.sh:213]: I think this means we'll use `GKE_CLUSTER_PSM_BASIC` clusters for fallback tests? Should we create new clusters for fallback?

Comment (eugeneo) [reply .kokoro/psm_interop_kokoro_lib.sh:213]: The test itself does not use GKE. This is only to satisfy the infra.

Comment (XuanWang-Amos) [reply .kokoro/psm_interop_kokoro_lib.sh:213]: I'm not familiar with this part of the framework, if we don't need GKE clusters, do we have to use `activate_gke_cluster` here?

Comment (eugeneo) [reply .kokoro/psm_interop_kokoro_lib.sh]: I do not know how to test this locally... I am now trying to bypass the cluster init. Please take another look.

Comment (XuanWang-Amos) [reply .kokoro/psm_interop_kokoro_lib.sh]: Skip them makes more sense, LGTM.

**Review by XuanWang-Amos — APPROVED** [review]

**Pull request merged**